.circleci/config.yml

```diff
@@ -123,7 +123,7 @@ memcached-container: &memcached-container
   image: memcached:1.5-alpine
 
 redis-container: &redis-container
-  image: redis:4.0-alpine
+  image: redis:6.2-alpine
```

Comment:
Redis 4 is not supported by Sidekiq 7. CircleCI needs to upgrade it's image to 6.2. That's fine, since 6.2 is the only supported version also for porta: https://access.redhat.com/articles/2798521